operations/mimir-mixin/dashboards/overview.libsonnet

```
@@ -32,7 +32,7 @@ local filename = 'mimir-overview.json';
     .addClusterSelectorTemplates()
 
     .addRow(
-      $.row('Mimir cluster health')
+      $.row('%(product)s cluster health' % $._config)
```

```suggestion
      $.row('%(product)s cluster health' % $._config.product)
```
Is this missing `.product` , I'm just looking for consistency, otherwise no idea

---

Using `$._config` is the correct approach. `%(product)s` is the expected to be a field of the object you pass (`$._config`).